app/Http/Controllers/RedirectsController.php

```diff
@@ -9,6 +9,7 @@ class RedirectsController extends Controller
 {
     public function __construct()
     {
+        $this->middleware('auth');
```

Comment:
Do we want both of these so that if someone isn't logged in they get a message to log in instead of a message that they must be an admin?

Reply:
Yep!